src/main.c

```diff
@@ -774,13 +777,13 @@ void main_imu_thread(void) {
 				}
 				float q_offset[4];
 				q_multiply(q, q3, q_offset);
-				tx_buf[0] = INT16_TO_UINT16(TO_FIXED_14(q_offset[0]));
-				tx_buf[1] = INT16_TO_UINT16(TO_FIXED_14(q_offset[1]));
-				tx_buf[2] = INT16_TO_UINT16(TO_FIXED_14(q_offset[2]));
-				tx_buf[3] = INT16_TO_UINT16(TO_FIXED_14(q_offset[3]));
-				tx_buf[4] = INT16_TO_UINT16(TO_FIXED_7(lin_ax)); //??? maybe something is wrong here?
-				tx_buf[5] = INT16_TO_UINT16(TO_FIXED_7(lin_ay));
-				tx_buf[6] = INT16_TO_UINT16(TO_FIXED_7(lin_az));
+				tx_buf[0] = TO_FIXED_15(q_offset[3]);
+				tx_buf[1] = TO_FIXED_15(q_offset[0]);
+				tx_buf[2] = TO_FIXED_15(q_offset[1]);
+				tx_buf[3] = TO_FIXED_15(q_offset[2]);
```

Comment:
Sorry, that was wrong order. It should `1230` order instead of `3012`. I will prepare a fix.